src/web/components/AgeWarning.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { css } from 'emotion';
+
+import ClockIcon from '@frontend/static/icons/clock.svg';
+
+import { from } from '@guardian/src-foundations/mq';
+import { palette } from '@guardian/src-foundations';
+import { textSans } from '@guardian/src-foundations/typography';
+import { visuallyHidden } from '@guardian/src-foundations/accessibility';
+
+const ageWarningStyles = (isSmall: boolean) => css`
+    ${isSmall ? textSans.xsmall() : textSans.medium()};
+    color: ${palette.neutral[7]};
+    background-color: ${palette.yellow.main};
+    display: inline-block;
+
+    > strong {
+        font-weight: bold;
+    }
+
+    padding: ${isSmall ? '3px 5px' : '6px 10px'};
```

Comment:
Minor: These variations feel very designery-tweaky. I wonder if we could use `vw` units to make things variable but without the magic px.

Reply:
Yeah, I agree. At the moment I'm just copying over from frontend but this means there is a lot of magic pixel dust. I think there's also scope for another PR here to review padding and spacing in general, ideally in tandem with @SiAdcock and `src-foundation`

Reply:
Our [spacing recommendations](https://zeroheight.com/2a1e5182b/p/075107) might help. Let me know if you'd like to discuss.